doc/build-windows.md

```diff
@@ -108,6 +108,7 @@ Acquire the source in the usual way:
     git clone https://github.com/bitcoin/bitcoin.git
 
 Once the source code is ready the build steps are below.
+cd bitcoin
```

Comment:
The indentation needs to match the other steps below for this to be formatted correctly :)

Reply:
@WhiteSymmetry If you need some help in understanding what needs to be done here, please feel free to ask.  (I agree with @MeshCollider's review and request.)

Reply:
Take a look at the rendered output: https://github.com/WhiteSymmetry/bitcoin/blob/527a0b805fb3d58ce57f86d795d2de677d1677ac/doc/build-windows.md#building-for-64-bit-windows  to see how this goes wrong.

Reply:
during review you can also quick "view" button in code changes section to see the formatting